tests/exploratory_analyses/feature_importance_test.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import pandas as pd
+from sklearn.neural_network import MLPClassifier
+from sklearn.preprocessing import StandardScaler
+
+from eis_toolkit.exploratory_analyses.feature_importance import evaluate_feature_importance
+
+feature_names = [
+    "Mag_TMI",
+    "Mag_AS",
+    "DRC135",
+    "DRC180",
+    "DRC45",
+    "DRC90",
+    "Mag_TD",
+    "HDTDR",
+    "Mag_Xdrv",
+    "mag_Ydrv",
+    "Mag_Zdrv",
+    "Pseu_Grv",
+    "Rd_U",
+    "Rd_TC",
+    "Rd_Th",
+    "Rd_K",
+    "EM_ratio",
+    "EM_Ap_rs",
+    "Em_Qd",
+    "EM_Inph",
+]
+
+X = pd.read_csv("../data/local/data/fake_smote_data.csv").to_numpy()
```

Comment:
If you put this dataset to somewhere in the data/remote, then the test will work for everyone. Now the dataset stays on your computer.

Reply:
I puty it in remote folder